curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c4f33bc3c3d8db7f732787.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 65c4f33bc3c3d8db7f732787
+title: Step 16
+challengeType: 20
+dashedName: step-16
+---
+
+# --description--
+
+Now, to perform the actual insertion, define an empty `insert` method within the `BinarySearchTree` class and give it a `self` parameter. Remember to use the `pass` keyword.
+
+# --hints--
+
+You should define an `insert` method with a `self` parameter within the `BinarySearchTree` class. Remember the `pass` keyword.
```

Comment:
this hint says pretty much the same thing as the instructions, an hint should be sufficently different to give an extra help